ArduPlane/Parameters.cpp

```diff
@@ -681,6 +681,13 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Standard
     GSCALAR(FBWB_min_altitude_cm,   "ALT_HOLD_FBWCM", ALT_HOLD_FBW_CM),
 
+    // @Param: ALT_FBWCM_MAX
+    // @DisplayName: Maximum altitude for FBWB and CRUISE modes
+    // @Description: This is the maximum altitude in meters that FBWB and CRUISE modes will allow. If you attempt to descend upper this altitude the plane will hold level. A value of zero means no limit.
+    // @Units: m
+    // @User: Standard
+    GSCALAR(FBWB_max_altitude_m,   "ALT_FBWCM_MAX", 0),
```

Comment:
variables do not have caps AFAIK....   think it should be fbwb_max_alt_m

Reply:
Hi! You mean a "previous" as ### k_param_FBWB_min_altitude_cm? 
Should I correct it too ?